### PR TITLE
Add release notes for Metals Radium release

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: website/blog/2019-06-11-radium.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+In this release we merged 24 PRs and closed 6 issues, full details:
+https://github.com/scalameta/metals/milestone/7?closed=1.
+
+## TL;DR
+
+- auto-import with gradle, maven and mill
+- better handling of requests that access the presentation compiler
+- code completions on keywords
+- VS code doesn't compile projects until it is focused
```
Comment (gabro) [inline website/blog/2019-06-11-radium.md]: I propose we pick a consistent term for vscode and stick to it. I propose "VS Code" (space after VS and capital C)

File: website/blog/2019-06-11-radium.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+- Metals is now a default server in the Sublime LSP package
+
+Check out the website and give Metals a try: https://scalameta.org/metals/
+
+## Auto-import with gradle, maven and mill
+
+Just as with `sbt` when you open a new project that contains scala code and is
+built with `Gradle`, `Maven` or `Mill`, a pop-up will show prompting you to
```
Comment (gabro) [inline website/blog/2019-06-11-radium.md]: The backticks seem unnecessary for tools names. I would use sbt, Gradle, Maven and Mill without backticks

File: website/blog/2019-06-11-radium.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+import the project. Once you click it the auto-import will start. There is more
+information about each of the build tools supported in the documentation.
```
Comment (gabro) [inline website/blog/2019-06-11-radium.md]: link to the documentation?

File: website/blog/2019-06-11-radium.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+## Better handling of requests to the presentation compiler
```
Comment (gabro) [inline website/blog/2019-06-11-radium.md]: This entire section seems a bit too focused on implementation details. I would rephrase it from a Metals user perspective and link to the relevant issue for the technical details.

Something like:

> Due to how requests to the presentation compiler worked, the Metals process would fail to exit properly in some cases. We've refactored the communication to make it more resilient, so now Metals should not be hanging around longer than necessary!

> As a bonus, this approach allows Metals to prioritize the most recently submitted request when choosing between competing hover/signatureHelp/completion requests.

File: website/blog/2019-06-11-radium.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+presentation compiler. This approach had several issues, of which most
+critically we risked blocking threads forever preventing the Metals process from
+exiting.
+
+Now, the APIs that access the presentation compiler return CompletableFuture and
+the implementation no longer uses .synchronized. This change had a nice
+side-effect that Metals now prioritizes the most recently submitted request when
+choosing between competing hover/signatureHelp/completion requests.
+
+## Code completions on keywords
```
Comment (gabro) [inline website/blog/2019-06-11-radium.md]: I would add a minor sentence as introduction here:

> Language keywords are now included in the auto-completions. For instance:

File: website/blog/2019-06-11-radium.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+}
+```
+
+will automatically give you suggestion to do:
+
+```scala
+object Main {
+    import
+}
+```
```
Comment (gabro) [inline website/blog/2019-06-11-radium.md]: > Keywords are suggested based on the context (e.g. you won't see `throw` suggested if the cursor is not inside a declaration)

File: website/blog/2019-06-11-radium.md [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+
+## VS code doesn't compile projects until it is focused
```
Comment (gabro) [inline website/blog/2019-06-11-radium.md]: See comment above about VS Code casing.

File: website/blog/2019-06-11-radium.md [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+
+When you edit your code outside the editor, like in case of switching git
+branches, VSCode will wait until it is focused again to start the compilation
+again. This helps to avoid uneccessary compilations ans strain on the processor.
```
Comment (gabro) [inline website/blog/2019-06-11-radium.md]:
```suggestion
again. This helps to avoid unnecessary compilations when you're not actively working on the code.
```

**Comment by tgodzik** [comment]
> Nice job, @tgodzik! 
> 
> I've left some suggestions. Also, I would add the bug fix for infinite recursion on completions that @olafurpg just submitted.

Thanks for checking and @olafurpg for fixing it up! :+1:

**Comment by olafurpg** [comment]
Ignoring flaky test failure, we need to split the slow/test CI job into smaller chunks.
